packages/overlay/src/components/Tabs.tsx

```diff
@@ -50,6 +63,7 @@ export default function Tabs({ tabs, nested }: Props) {
             <NavLink
               to={`${nested ? '' : '/'}${tab.id}`}
               key={tab.id}
+              replace={true}
```

Comment:
Oh I see this actually solves the problem I mentioned in my review, right?
So we just need to apply it to the respective places

Reply:
Yeah, might be a bit more delicate there - tried it quickly and didn't work ootb

We could also consider hitting `ESC` in quick succession closes Spotlight.

Reply:
Instead of this, maybe we add another key combination for dedicated open/close functionality? #158